code/parachain/frame/vesting/src/lib.rs

```diff
@@ -606,7 +610,7 @@ impl<T: Config> Pallet<T> {
 
 					claims_per_schedule
 						.try_insert(*id_to_claim, available_amount)
```

Comment:
This `expect` is ok IMO as there is no chance(invariant) of it having a `MaxVestingSchedulesExceeded` given it is reinserting a value that was read in this same code.

Reply:
Yeah, but I was suggested not to have any `expect` besides the `GenesisBuild`, and I guess it doesn't hurt to remove it right?